samples/cellular/lwm2m_client/src/ipso/ipso_colour_sensor.c

```diff
@@ -30,7 +30,7 @@ LOG_MODULE_DECLARE(app_sensors, CONFIG_APP_LOG_LEVEL);
 #else
 #define COLOUR_VERSION_MINOR 0
 #define NUMBER_OF_OBJECT_FIELDS 3
-#endif /* ifdef CONFIG_LWM2M_IPSO_APP_COLOUR_VERSION_1_1 */
+#endif /* ifdef CONFIG_LWM2M_IPSO_APP_COLOUR_SENSOR_VERSION_1_1 */
```

Comment:
Why are we keeping these extra strings? Since this PR is aiming for a cleanup, I would argue it makes sense to align everything to the same style.

```suggestion
#endif /* CONFIG_LWM2M_IPSO_APP_COLOUR_SENSOR_VERSION_1_1 */
```